play-json/jvm/src/main/scala/play/api/libs/json/EnvWrites.scala

```diff
@@ -134,14 +141,35 @@ trait EnvWrites {
    * The default typeclass to write a `java.time.Instant`,
    * using '2011-12-03T10:15:30Z' format.
    */
-  implicit val DefaultInstantWrites = new Writes[Instant] {
-    def writes(i: Instant): JsValue = JsString(i.toString)
+  implicit val DefaultInstantWrites: Writes[Instant] =
+    Writes[Instant] { i => JsString(i.toString) }
+
+  /**
+   * The default typeclass to write a `java.time.LocalTime`,
+   * using '10:15:30' format.
+   */
+  implicit val DefaultLocalTimeWrites: Writes[LocalTime] =
+    temporalWrites[LocalTime, DateTimeFormatter](DateTimeFormatter.ISO_TIME)
+
+  /**
+   * Serializer for `java.time.LocalTime` as JSON number.
+   *
+   * {{{
+   * import java.time.LocalTime
+   * import play.api.libs.json.Writes
+   *
+   * implicit val ltnWrites = Writes.LocalTimeNumberWrites
+   * }}}
+   */
+  val LocalTimeNumberWrites: Writes[LocalTime] = Writes[LocalTime] { t =>
```

Comment:
> LocalTime doesn't use milliseconds epoch but nanos of day

It just seemed odd to me that writing a `LocalDate` as a number would use different units than `LocalTime`.

Can we just rename the reads/writes to specify the units they are using. e.g. `LocalDateNanosOfDayWrites`, `LocalTimeMillisWrites`. I'm ok with either that or taking a `TimeUnit` as a parameter.